packages/office-ui-fabric-react/src/components/Callout/CalloutContent.tsx

```diff
@@ -336,4 +341,25 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
       this._targetWindow = getWindow();
     }
   }
+
+  private _setHeightOffsetEveryFrame(): void {
+    if (this._calloutElement && this.props.finalHeight) {
+      this._setHeightOffsetTimer = this._async.requestAnimationFrame(() => {
+        const calloutMainElem = this._calloutElement.firstChild as HTMLElement;
```

Comment:
Dunno how expensive this is (probably not at all), but it could be moved outside the requestAnimationFrame.